app/classifier/tasks/flexible-survey/utility.coffee

```diff
@@ -0,0 +1,8 @@
+module.exports =
+  getQuestionIDs: (task, choiceID) ->
+    return [] if task.choices[choiceID].noQuestions
+    return task.questionsOrder unless task.questionsMap? and choiceID of task.questionsMap
+    task.questionsMap[choiceID]
+  getQuestions: (task, choiceID) ->
+    @getQuestionIDs(task, choiceID).map (idx ->
```

Comment:
I think this function is unused, but if I'm wrong `idx` would need to be wrapped in parens